avocado/core/test.py

```diff
@@ -782,7 +782,11 @@ def skip(self, message=None):
         :param message: an optional message that will be recorded in the logs
         :type message: str
         """
-        raise exceptions.TestSetupSkip(message)
+        dep_msg = '[WARNING: self.skip() will be deprecated, please ' \
+                  'use self.cancel()] '
+        if message is not None:
+            dep_msg += message
```

Comment:
Well message could be anything, I'd suggest using `"%s %s" % (dep_msg, message)`.